### PR TITLE
PERF-210 PERF-222 consent & consent-ontology

### PR DESCRIPTION
**Pull request by gmalkov**

### CODE REVIEW

File: profiles/consent-state/terraform/consent/cloudsql-init.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
+resource "google_sql_ssl_cert" "client_cert" {
```
Comment (RLuckom) [inline profiles/consent-state/terraform/consent/cloudsql-init.tf]: Do you think this is a candidate for becoming a module? (not in this PR)

File: profiles/consent-state/terraform/consent/elasticsearch-init.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+resource "null_resource" "elasticsearch_init" {
+  triggers = {
+    always = "${uuid()}"
+  }
+  provisioner "local-exec" {
+    command = <<EOF
+curl -X PUT "http://${var.elasticsearch_hostname}:9200/ontology/" -H 'Content-Type: application/json' -d '{"settings":{"index":{"number_of_replicas":1}}}'
```
Comment (RLuckom) [inline profiles/consent-state/terraform/consent/elasticsearch-init.tf]: can you make `number_of_replicas` a variable?

File: profiles/agora-mongo/terraform/agora-mongo/agora-mongo.tf [hunk 3]
```diff
@@ -10,6 +10,7 @@ module "mongodb" {
   owner                    = "${var.owner}"
   service                  = "${var.service}"
   instance_name            = "${var.service}"
+  mongodb_image_tag        = "3.6.14"
```
Comment (RLuckom) [inline profiles/agora-mongo/terraform/agora-mongo/agora-mongo.tf]: Can you make this a variable in the profile?

File: profiles/consent-mongo/terraform/consent-mongo/consent-mongo.tf [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  project                  = "${var.google_project}"
+  owner                    = "${var.owner}"
+  service                  = "${var.service}"
+  instance_name            = "${var.service}"
+  mongodb_service_account  = "${var.config_reader_service_account}"
+  mongodb_roles            = var.mongodb_roles
+  mongodb_app_username     = "${var.service}"
+  mongodb_app_password     = "${random_id.mongodb-user-password.hex}"
+  mongodb_root_password    = "${random_id.mongodb-root-password.hex}"
+  mongodb_database         = "${var.service}"
```
Comment (RLuckom) [inline profiles/consent-mongo/terraform/consent-mongo/consent-mongo.tf]: Add variable for mongo version?

File: profiles/consent-state/terraform/consent/outputs.tf [hunk 5]
```diff
@@ -0,0 +1,28 @@
+output "ATTENTION" {
+  value = <<EOF
+THIS PROFILE REQUIRES MANUAL STEPS!
+The manual steps are required to initialize the mysql database.
+To run the manual steps run the script in the profile root:
+python manaul_steps.py
```
Comment (RLuckom) [inline profiles/consent-state/terraform/consent/outputs.tf]: "manual"

File: broad-wb-rluckom.json [hunk 6]
```diff
@@ -331,6 +331,28 @@
...
+    "consent-configs": {
+      "env": {
+        "APP_NAME": "consent",
+        "FIRECLOUD_DEVELOP_GIT_BRANCH": "perf-210-consent",
+        "SERVICE_GIT_SHA_12_CHAR": "fead1cb7703e"
+      },
+      "terraform": {
+        "copy_secrets": {
+          "secret_keys": {
+            "from_path": "secret/dsde/firecloud/perf/consent/consent.conf",
```
[the remark this answers is not in the document]
Comment (gmalkov) [reply broad-wb-rluckom.json]: Yeah I think it's just for the mailconfig stuff

File: profiles/consent-state/terraform/consent/cloudsql-init.tf (hunk 1 above)
Comment (gmalkov) [reply profiles/consent-state/terraform/consent/cloudsql-init.tf]: Mmm sure I can see that

File: profiles/consent-state/terraform/consent/elasticsearch-init.tf (hunk 2 above)
Comment (gmalkov) [reply profiles/consent-state/terraform/consent/elasticsearch-init.tf]: Oh yeah I was actually gonna do that and then forgot

**Pull request merged**